src/writer/launchdarkly_client.py

```
@@ -0,0 +1,246 @@
+            if org_id is not None:
+                context_builder.set("organizationId", org_id)
+
+            if mode:
+                context_builder.set("mode", mode)
+
+            if writer_application and writer_application.get("id"):
+                context_builder.set("writerApplicationId", writer_application["id"])
+
+            return context_builder.build()
+
+        except Exception as e:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)